integration_tests/src/main/python/json_matrix_test.py

```diff
@@ -1257,7 +1257,11 @@ def test_from_json_string_arrays(std_input_path, input_file):
     "single_quoted_strings.json",
     "boolean_formatted.json",
     "int_array_formatted.json",
-    pytest.param("int_struct_formatted.json", marks=pytest.mark.xfail(condition=is_before_spark_342(),reason='https://github.com/NVIDIA/spark-rapids/issues/10588')),
+    "int_struct_formatted.json",
```

Comment:
Can you please update the other tests to include int_struct_formatted_problematic_rows.json?  The idea is that we have several different schemas, several different operators, and several different test files. We want all of the the operators and all of the schemas to run against all of the files. Even if they include errors at least it will let us see that we are handling it the same way that Spark is.

Reply:
Oh, shoot. Yes, sorry I missed that.

Reply:
I've added the new test file to all affected tests.